testdata/custom_project_path_app/plan.toml

```diff
@@ -0,0 +1,5 @@
+[[requires]]
```

Comment:
Should this `testdata` directory be merged with `integration/testdata` directory?

Reply:
Or if it's not used in integration tests, maybe rename the directory to `samples` or `examples`.

Reply:
Seems like a duplicate directory that we used when we first started developing the integration tests and forgot to move it under integration directory. Removed it https://github.com/paketo-community/ubi-nodejs-extension/pull/9/commits/898816030299871741ac44d39d47163e72b139de